Core/Core/Logging/SpeckleLog.cs

```diff
@@ -271,23 +292,16 @@ private static void AddUserIdToGlobalContextFromDefaultAccount()
     }
     catch (Exception ex) when (!ex.IsFatal())
     {
-      Logger.Warning(ex, "Cannot set user id for the global log context.");
+      // To log it after Logger initialized as deferred action.
+      s_isMachineIdUsed = true;
     }
-    GlobalLogContext.PushProperty("id", id);
-
-    SentrySdk.ConfigureScope(scope =>
-    {
-      scope.User = new User { Id = id };
-    });
+    return id;
```

Comment:
Should `s_isMachineIdUsed` also be added when `defaultAccount == null` (opposite condition than line 288)